src/pages/ProductDetail.js

```diff
@@ -1,71 +1,86 @@
 import { useParams } from "react-router-dom";
-import { useState } from "react";
+import { useState, useContext } from "react";
 import { Swiper, SwiperSlide } from "swiper/react";
 import Card from "../components/common/Card";
 import StyledFlex from "../components/common/StyledFlex";
 import ItemCounter from "../components/common/ItemCounter";
+import ShoppingCartContext from "../utils/state/ShoppingCartContext";
 import { useProduct } from "../utils/hooks/useProduct";
 import 'swiper/swiper-bundle.min.css'
 import 'swiper/swiper.min.css'
 const ProductDetailPage = () => {
```

Comment:
After adding the total stock available of a product, it could be really useful if the button becomes disabled.
<img width="1647" alt="Screen Shot 2022-05-17 at 16 14 04" src="https://user-images.githubusercontent.com/96157153/168911230-d5caa88c-d738-431a-9863-f37d3448d1a5.png">